unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAnimatorLegacy.cs

```diff
@@ -109,6 +108,14 @@ public class BlackBoard
     private bool isUpdateRegistered = false;
 
     private Ray rayCache;
+    private bool hasTarget;
+
+    private void Awake()
+    {
+        hasTarget = target != null;
+        if (!hasTarget)
+            Debug.LogError(message: $"Target is not assigned. {nameof(UpdateInterface)} will work incorrectly.", this);
```

Comment:
Just a minor grammatical thing, instead of "will work incorrectly" maybe "will not work correctly" is better